WIP: Minor fixes and update to new ink! version

athei opened the pull request:
I am currently working through the contracts workshop as part of my onboarding. I am just fixing some minor oversights that I notice while doing that.

Robbepop commented on 1/storing-a-mapping.md:
```diff
@@ -114,7 +114,7 @@ mod mycontract {
         #[ink(message)]
         fn get_my_number(&self) -> u32 {
-            let caller = env.caller();
+            let caller = self.env.caller();
```
```suggestion
            let caller = self.env().caller();
```

athei replied:
Yep good spot.

athei commented on 1/incrementing-the-value.md:
````diff
@@ -43,7 +43,7 @@ impl MyContract {
 ```
 
-However, if the value is not initialized, your contract will compile fine, but will panic during contract execution! **We really cannot understate how easy it is to make mistakes this way.**
+However, if the value is not initialized, your contract will compile fine, but will panic during contract execution! **We really cannot overstate how easy it is to make mistakes this way.**
````
Now that I think about it, it seems to be actually correct. I think I got it backwards.

athei commented:
> Thank you for the fixes. However, I guess there are many more things that need to be fixed for our ink! workshop.

One step at a time I guess. Right now, I only intend on fixing things that somehow blocked or confused me while doing the workshop.

Robbepop commented:
We can merge as soon as you press "Ready for review". xD

athei commented:
I guess I will put the coming changes into another PRs then. I just removed the commit regarding the `overstate` vs `understate` and squashed the fixup. Yea just merge this micro pr when you are ready.

Robbepop commented:
Thank you for your help in keeping this updated!
